src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -121,7 +121,15 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
 
         if (read !== null) {
             this.profileImportFilePath = files[0];
-            this.profileImportContent = await ProfileUtils.parseYamlToExportFormat(read);
+            try {
+                this.profileImportContent = await ProfileUtils.parseYamlToExportFormat(read);
+            } catch (e: unknown) {
+                if(e instanceof R2Error) {
```

Comment:
Well probably want to catch all errors here, including the once possibly thrown by `yaml.parse` or any future additions to the helper function. So I would recommend skipping the `if` check and just using `R2Error.fromThrownValue`.